.github/workflows/cpp-packaging.yml

```diff
@@ -601,45 +604,75 @@ jobs:
           firebase-cpp-sdk-android-c++-package
         failOnError: false
 
-  run_integration_tests:
-    name: run-integration-tests
+  tests:
     needs: [merge_packages, download_sdk_package]
-    if: always()
-    runs-on: ubuntu-latest
+    if: github.event.inputs.runIntegrationTests != 0 && !cancelled()
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        target_platform: [Desktop, Android, iOS]
+        exclude:
+          - os: ubuntu-latest
+            target_platform: iOS
+          - os: windows-latest
+            target_platform: iOS
+      fail-fast: false
+    env:
+      apis: admob,analytics,auth,database,dynamic_links,firestore,functions,instance_id,messaging,remote_config,storage
+      android_device: flame
+      android_api: 29
+      ios_device: iphone11pro
+      ios_version: 13.3
     steps:
-      - name: setup Xcode version (macos)
-        if: runner.os == 'macOS'
-        run: sudo xcode-select -s /Applications/Xcode_${{ env.xcodeVersion }}.app/Contents/Developer
-
-      - name: download SDK zip
-        uses: actions/download-artifact@v2
-        with:
-          name: firebase_cpp_sdk.zip
-          path: .
-
-      - name: download hash
```

Comment:
Was the hash check step purposefully removed?

Reply:
Yes, the hash command is not present on all operating systems.